yapapi/props/builder.py

```diff
@@ -7,8 +7,37 @@
 
 from . import Model
 
+# TODO in 0.4+: `cons` is not obvious, should be named `constraints`
+# TODO in 0.4+: maybe `props` should just be named `properties` (?)
```

Comment:
I agree with you. Moreover, `props` can be read as `properties` or `proposals`.
And cons (in Lisp) is: https://en.wikipedia.org/wiki/Cons

Reply:
well, it also can be interpreted as "arguments to the contrary" as in "pros and cons" ... especially with "props" being so similar to "pros" ...

Reply:
Props to that change!